app/views/home/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="row">
+  <div class="col-lg-12">
+    <div class="row">
+      <div class="col-md-4">
+        <h1>Top Movies</h1>
+        <% @movies.by_ranking.first(10).each do |movie| %>
```

Comment:
It's relatively minor, but this might be a good place to use a view helper method. We could create a method in `app/helpers/home_helper.rb` called `top_movies`:
```ruby
def top_movies
  Movie.all.by_ranking.first(10)
end
```
and then just use `<% top_movies.each do |movie| %>` here. While it doesn't DRY out the code much, it does help a bit to move the logic out of the view code, and insulates it from having to make the decision about exactly how the top ten movies/books/albums are selected.